conda-store-server/tests/conftest.py

```diff
@@ -24,9 +24,13 @@ def conda_store_config(tmp_path, request):
 
     filename = pathlib.Path(tmp_path) / "database.sqlite"
```

Comment:
Not essential, but needs `".conda-store"` here, to be consistent with how we run this outside of tests.